oeps/best-practices/oep-0067-bp-tools-and-technology.rst

```diff
@@ -181,6 +181,21 @@ Frontend Technology Selection
    Documentation on how to configure Renovate automation on a repository is available
    in the `Upgrade Automation How-to`_.
 
+#. **JavaScript projects should use caret ranges for npm dependencies**
+
+   **Rationale**: Using caret ranges (e.g., ``"^1.2.3"``) in our package.json more clearly
+   communicates that our projects are designed to accept semver-compatible (minor and patch)
+   updates. This intent is important because merely pinning top-level dependencies does not
+   extend to their transitive dependencies—potentially leading to duplicative or outdated
+   packages in our bundles. Moreover, specifying dependencies with caret ranges streamlines
+   our Renovate workflow. With caret ranges, non-breaking updates are typically resolved
```

Comment:
The previous section already has links set up for Renovate so we can utilize that here too!

```suggestion
   our `Renovate`_ workflow. With caret ranges, non-breaking updates are typically resolved
```